stdimage/management/commands/rendervariations.py

```diff
@@ -8,6 +9,8 @@
 
 from stdimage.utils import render_variations
 
+logger = logging.getLogger()
```

Comment:
I explain further down why I don't think using a logger is a good idea. But as a side note here:
I would recommend to use the package name. That way people have only a single logger per package, that they can configure in their projects.

Reply:
I also don't know the default, I think you are getting the root logger here, where I think you wanted `__name__`. Anyhow, check my comment further down about `stderr`.

Reply:
yeah u are right, I just missed that.
*quality code*